google/cloud/pubsub/samples/subscription_admin_samples.cc

```diff
@@ -166,6 +168,8 @@ void CreateCloudStorageSubscription(
 void CreateDeadLetterSubscription(
     google::cloud::pubsub_admin::SubscriptionAdminClient client,
     std::vector<std::string> const& argv) {
+  //! [dead-letter-create-subscription]
```

Comment:
Here and elsewhere, I thought you decided not to include doxygen regions for these samples?

https://github.com/googleapis/google-cloud-cpp/pull/13527#discussion_r1474881559

Reply:
I'm confused what the difference is.

I added the tags so I can point them to the write files in google3.

Reply:
START -> devrell (CS)
other references within our code (in doxygen for the client comments)

Reply:
Ack, now I understand the difference.

Removed all doxygen since this is a generated library and kept the devreel tags.